docs/UserGuide.md

```diff
@@ -40,95 +50,163 @@ Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a task: `add`
+Adds a task to GGist<br>
+General format: `add TASK, [DATE], [TIME], [PRIORITY], [FREQUENCY]`
 
-> Persons can have any number of tags (including 0)
+<img src="images/2ndimpt.png" width="500"><br>
+
+###### With deadline
+Format: `add TASK, DATE, TIME, [PRIORITY], [FREQUENCY]`
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `add write diary, jul 10, 1300`
+* `add prepare presentation slides, mon, 1400, high`
+
+###### Without any deadline
+Format: `add TASK, [DATE], [TIME], [PRIORITY], [FREQUENCY]`
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+Examples: 
+* `add buy milk, low`
+* `add buy present for brother's birthday`
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+###### With start and end time
```

Comment:
this and the deadline seems very similar (except the time format); so, may be you combine them to one?